apps/builder/app/builder/shared/commands.ts

```diff
@@ -504,6 +504,18 @@ export const { emitCommand, subscribeCommands } = createCommandsEmitter({
       name: "wrapInLink",
       handler: () => wrapIn("Link"),
     },
+    {
+      name: "wrapInButton",
+      handler: () => wrapIn("Button"),
+    },
```

Comment:
Does your designer really want to wrap with buttons? Caz in webflow buttons are actually links which can be very misleading.

Reply:
Yeah he understands the difference. In webstudio, buttons are buttons, right?

Reply:
Yep